apps/extension/package.json

```diff
@@ -47,6 +47,7 @@
     "bignumber.js": "^9.1.1",
     "buffer": "^6.0.3",
     "dompurify": "^3.0.2",
+    "fp-ts": "^2.16.9",
```

Comment:
Wanted to add this before I forget - it previously got it's dependency as it was specified in Namadillo. Makes it explicit here